docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/LoadImages.cs

```diff
@@ -36,7 +36,7 @@ public static void Example()
 
             var imagesFolder = Path.GetDirectoryName(imagesDataFile);
             // Image loading pipeline. 
-            var pipeline = mlContext.Transforms.LoadImages(imagesFolder, "ImageReal", "ImagePath");
+            var pipeline = mlContext.Transforms.LoadImages("ImageReal", imagesFolder, "ImagePath");
```

Comment:
>LoadImages [](start = 48, length = 10)

Hi @rogancarr, thanks for addressing this issue.
Sorry, I should have been more specific in the issue itself - there are a few more references to this API in the samples project (DnnFeaturizeImage, ResizeImage, ConvertToGrayScale, ExtractPixels), could you update these as well?